.github/workflows/ci.yml

```diff
@@ -816,6 +842,66 @@ jobs:
           fail_on_unmatched_files: true
           prerelease: ${{ contains(steps.semver.outputs.group1, '-') }}
 
+  upload-sentry:
+    name: sentry upload
+    if: ${{ startsWith(github.ref, 'refs/tags/v') }}
+    needs:
+      - build
+      - build-linux
+      - copyright
+      - dartanalyze
+      - dartdoc
+      - dartfmt
+      - docker-push
+      - test-e2e
+      - test-unit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: build-web-${{ github.run_number }}
+      - run: mkdir -p build/web/
+      - run: unzip *.zip -d build/web/
```

Comment:
@SleepySquash why do we need web build here?

Reply:
@tyranron, to retrieve the source maps (`main.dart.js.map` file) web compilation splits out when built with `--source-maps` flag. It is used by Sentry to decipher the logs.

Actually, now that you've mentioned it, I think this file might be uploaded as a separate artifact (`debug-web-...` or so) and then fetched here, instead of the whole web build.